x/stakeibc/keeper/ibc_handlers.go

```diff
@@ -24,6 +24,7 @@ import (
 // TODO(TEST-33): Scope out what to store on different acks (by function call, success/failure)
 func (k Keeper) HandleAcknowledgement(ctx sdk.Context, modulePacket channeltypes.Packet, acknowledgement []byte) error {
 	ack := channeltypes.Acknowledgement_Result{}
+	packetSequenceKey := string(modulePacket.Sequence)
```

Comment:
can we add some error checking here e.g. if sequence is nil or empty, break? worried about reliability with these packet sequence #s

Reply:
Because it's an int field on a proto, it'll always come back as 0 or some positive int. 0 is technically valid (will just be the first packet), so not sure what the best way to check this is. Wdyt?

Reply:
hmm fair points - no good way to validate it afaiu